i18n/en.toml

```diff
@@ -27,9 +27,30 @@ other = "Yes"
 [feedback_no]
 other = "No"
 
-[latest_version]
+[shortcode_latest_version]
 other = "latest version."
 
+[shortcode_version_check_mustbe]
+other = "Your Kubernetes server must be version "
+
+[shortcode_version_check_mustbeorlater_start]
+other = "Your Kubernetes server must be version "
+
```

Comment:
Hello @seokho-son, a couple of thoughts:
- Are the two keys above assigned to the same string?
- I am not sure about prefixing the keys with `shortcode` or `main`. How do these prefixes help to identify or discriminate the keys? The keys could be differentiated if there is more than one value or variation on a given key. The names of the keys should be short and just indicate the actual text.
- I am not a big fan of adding punctuation into the string values, unless the punctuation needs to be translated or if there is more than one version of a key such as `latest version ` versus `latest version.`.

Reply:
@kbhawkey Thank you for your comments ! 

- Are the two keys above assigned to the same string?
**[replay]** Yes. but it is intentional because of translation difficulty. Even though the strings are same, they can be translated differently according to position and structure of sentence. 
For instance, 
[shortcode_version_check_mustbe] can be translated into "쿠버네티스 서버는 다음 버전이어야 함. 버전" and 
[shortcode_version_check_mustbeorlater_start] can be translated into "쿠버네티스 서버는 버전 "

- I am not sure about prefixing the keys with shortcode or main. How do these prefixes help to identify or discriminate the keys? The keys could be differentiated if there is more than one value or variation on a given key. The names of the keys should be short and just indicate the actual text.
**[replay]** I understand your concerns. I think I used *.toml in unconventional way.  
Usually, the prefixed English words in Layout files are not fully structured sentences, and we need to replace a key with whole string as is for a complete translation. 
So, I think the key may not be used in other documents except the files in Layout. 
The reason I placed the `shortcode_<related file name>` prefix is not for discrimination. It is for traceability to help contributors when they want to change any strings in Layout files.
If necessary, I think we can remove the prefix, and add an annotation in *.toml to separate sections for the keys used in shortcodes.
I am not sure about the `main` prefix. It is not a part of this PR, but I think it can be removed.

- I am not a big fan of adding punctuation into the string values, unless the punctuation needs to be translated or if there is more than one version of a key such as latest version versus latest version..
**[replay]** I agree with you. I wish to reduce punctuation. However, for this moment, shortcodes includes several prefixed English words which is hard to be modified by i18n teams, and utilizing *.toml in translating remnant partials in templates and shortcodes was agreed in sig-docs meeting with consideration of management efficiency.

Please let me know if you have any further idea or requests regarding my reply. Thank you :)

Reply:
@seokho-son 👋 

> [replay] Yes. but it is intentional because of translation difficulty. Even though the strings are same, they can be translated differently according to position and structure of sentence.

Thanks for this clarification. Please add a comment indicating that this string is duplicated specifically for Korean, so that other languages know that they don't need to duplicate it also.

> If necessary, I think we can remove the prefix, and add an annotation in *.toml to separate sections for the keys used in shortcodes.

This is my preferred solution. Adding the `shortcode_` prefix inconsistently seems to create more confusion than it solves.

Reply:
@kbhawkey @zacharysarah To resolve your concerns, I updated PR as follows,
1. Removed one of the keys which have the same string. Instead, I rephrased the original sentence that causes Korean translation difficulty.
``` 
In en.toml

[version_check_mustbe]
other = "Your Kubernetes server must be version "

[version_check_mustbeorlater]
other = "Your Kubernetes server must be or later than version "
```
``` 
In version-check.html

{{ if eq $minVersion (.Page.Param "version")  }}
{{ T "version_check_mustbe" }}{{ $minVersion }}.
{{ else if $minVersion}}
{{ T "version_check_mustbeorlater" }}{{ $minVersion }}.
```

2. Removed `shortcode_` prefix as suggested.

Thank you :)